### PR TITLE
fix: safe bottom padding

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- There is a known safe bottom space natively which is defined at the footer CSS module.
- The difference between previous implementations was the usage of it rather than the explicit `pb-6` class.

Also tested on simulator:

Screenshot 2024-04-08 at 8 03 32 PM

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-256 #done

### Preview domain
https://mi-256-native-footer.preview.app.daily.dev

### CODE REVIEW

**Review by nensidosari — APPROVED** [review]
Looks good, can't see the deployment though. 

Would suggest whenever this is merged to let design team know if they want to check this particular thing out since it brought a lot of reported issues.

**Review by capJavert — APPROVED** [review]

File: packages/webapp/components/footer/FooterNavBarV1.tsx [hunk 1]
```diff
@@ -58,7 +58,7 @@ const Tab = ({ tab, isActive }: TabProps) => {
   return (
     <FooterNavBarItem
       isActive={isActive}
-      className="flex h-full flex-col items-center justify-center"
+      className="flex h-full flex-col items-center justify-center py-2"
```
Comment (kopancek) [inline packages/webapp/components/footer/FooterNavBarV1.tsx:61]: Why do we need the `py-2` here?

Comment (kopancek) [reply packages/webapp/components/footer/FooterNavBarV1.tsx:61]: Let's make sure there's no conflict with my changes here: https://github.com/dailydotdev/apps/pull/2950 :)

Comment (sshanzel) [reply packages/webapp/components/footer/FooterNavBarV1.tsx:61]: Looks good. Thank you :)

**Pull request merged**